### ISSUE TITLE
Unable to replicate results from EmotionNet 

### ISSUE THREAD

**Issue by ivona-tooploox** [co60ca/EmotionNet2#3]
Hi,  I loaded your pretrained model and tested a few images and I get results quite different from what was reported for first version (https://github.com/co60ca/EmotionNet/blob/master/README.md).

Is it expected? 

![screen shot 2018-02-02 at 12 17 08](https://user-images.githubusercontent.com/32739893/35730623-0c314ac8-0813-11e8-803d-b796698984b2.png)

**Comment by co60ca** [comment]
Hi, yes! This model was trained differently from EmotionNet(1) and the weights are therefore different. This model should generalize better, meaning more random photos should work. EmotionNet(1) had the issue that random photos often did not classify correctly.

In my report for this project there was a confusion matrix and as I recall Afraid and surprised often missclassified for one another. I did not test this image you have provided in EmotionNet2.

Closing, as expected. If you want higher probabilities you will need to find more training data.

(Also consider asking some friends if they think this is surprised or afraid, you will probably get some of both)
